spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :feature do
+  describe 'Post Index' do
+    background do
```

Comment:
- [ ] Please I am unable to run the tests because of this issue in the screenshot, please kindly checkout what the issue is and fix it.
![failingTests](https://user-images.githubusercontent.com/62849872/162260461-68730a1d-4652-4b9d-b0e7-bdc7f18e6f51.png)